backend/tracim_backend/lib/core/mention.py

```diff
@@ -164,8 +165,12 @@ def get_receiver_ids(
         else:
             # send to mentioned user
             user_api = UserApi(session=session, config=config, current_user=None)
-            user = user_api.get_one_by_username(recipient)
-            return [user.user_id]
+            try:
+                user = user_api.get_one_by_username(recipient)
+                return [user.user_id]
+            except UserDoesNotExist:
+                logger.warning(cls, "user of username {} could not been found !".format(recipient))
```

Comment:
This log wont help to investiguate because it does not explain why there is an error

Reply:
updated